src/parser.c

```diff
@@ -235,45 +286,66 @@ struct l_queue *parser_update_to_list(json_object *jso)
 	int jtype;
 	int olen;
 	uint8_t sensor_id;
+	bool err;
```

Comment:
I prefer the variable name has_err which was used in others functions such as parser_data_create_object function.

Reply:
i agree